docs/stylesheets/language-tags.css

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Supported language tag styling  */
+
+.language-support-tag {
+ display: inline-block;
+ font-family: Roboto-Regular, Roboto, Arial, sans-serif;
+ font-size: 0.6em;
+ font-weight: 400;
+ line-height: 1em;
+ color: white;
+ border-radius: 0.2em;
+ overflow: hidden; /* Ensures the border-radius is applied to the child elements */
+}
+.supported {
+ background-color: #555555;
+ padding: 4px 6px;
+ display: inline-block;
+}
+.python {
+ background-color: #007ec6;
+ padding: 4px 6px;
+ display: inline-block;
+}
+.java {
+ background-color: #ED8B00;
+ color: black;
+ padding: 4px 6px;
+ display: inline-block;
+}
+.go {
+ background-color: #00ADD8;
+ color: black;
+ padding: 4px 6px;
+ display: inline-block;
+}
```

Comment:
For some reason this CSS selector is conflicting with and changing the bg color for code that is syntax highlighted as `console`:

<img width="1914" height="1210" alt="Screenshot 2025-11-03 at 10 42 52 AM" src="https://github.com/user-attachments/assets/b35f5919-71e4-4ff6-a1ad-4885f60c7144" />

Reply:
good catch. thanks.

I renamed all the css classes to "lst-*" to avoid the class name conflicts like this.